src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,68 @@
+import classNames from 'classnames';
+import { FilterStatus } from '../../enums/Status';
+
+type Props = {
+  activeFilter: FilterStatus,
+  setFilter: React.Dispatch<React.SetStateAction<FilterStatus>>,
+};
+
+export const Footer: React.FC<Props> = ({ setFilter, activeFilter }) => {
+  const handleClick = (event: React.MouseEvent) => {
+    const option = event.currentTarget.textContent;
+
+    setFilter(option as FilterStatus);
+  };
+
+  return (
+    <footer className="todoapp__footer" data-cy="Footer">
+      <span className="todo-count" data-cy="todosCounter">
+        4 items left
+      </span>
+
+      <nav className="filter" data-cy="Filter">
+        <a
+          data-cy="FilterLinkAll"
+          href="#/"
```

Comment:
it might not work, the link has to start from slash /
```suggestion
          href="/#"
```